src/sentry/notifications/utils/__init__.py

```diff
@@ -526,6 +527,11 @@ def _find_span_by_id(self, id: str) -> Dict[str, Any] | None:
                 return span
         return None
 
+    def get_span_duration(self, span: Dict[str, any]) -> timedelta:
+        return timedelta(seconds=span.get("timestamp", 0)) - timedelta(
+            seconds=span.get("start_timestamp", 0)
+        )
```

Comment:
Totally fine but maybe a little strange? Why initialize two separate `timedelta` objects?
```suggestion
        return timedelta(seconds=span.get("timestamp", 0) - span.get("start_timestamp", 0))
```

Reply:
Good call, no reason, I like what you suggested!